scanner/src/main/kotlin/storages/LocalFileStorage.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2019 Bosch Software Innovations GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.scanner.storages
+
+import ch.frankel.slf4k.*
+
+import com.here.ort.model.Identifier
+import com.here.ort.model.ScanResult
+import com.here.ort.model.ScanResultContainer
+import com.here.ort.model.readValue
+import com.here.ort.model.yamlMapper
+import com.here.ort.utils.log
+import com.here.ort.utils.safeMkdirs
+import com.here.ort.utils.showStackTrace
+
+import java.io.File
+import java.io.IOException
+
+class LocalFileStorage(
```

Comment:
Does this class need thread safety?

Reply:
Yes, I believe so. Thanks for reminding me. Done.